docs/en/latest/plugins/log-rotate.md

```diff
@@ -39,6 +39,7 @@ You can configure how often the logs are rotated and how many logs to keep. When
 |--------------------|---------|----------|---------|------------------------------------------------------------------------------------------------|
 | interval           | integer | True     | 60 * 60 | Time in seconds specifying how often to rotate the logs.                                       |
 | max_kept           | integer | True     | 24 * 7  | Maximum number of historical logs to keep. If this number is exceeded, older logs are deleted. |
+| max_size           | integer | False    | -1      | Max size(Bytes) of log files to be rotated, size check would be skipped with a value less than 0. |
```

Comment:
Should we clarify that rotating the logs when time is up or when the log size reaches max_size?

Reply:
done